### ISSUE TITLE
Import Problem - SalesOrderLineItem has no order

### ISSUE THREAD

**Issue by bhotrock** [inventree/InvenTree#6261]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

When trying to import a CSV file with Sales Order Line Items, I receive an error:  
Line number: 1 - SalesOrderLineItem has no order.

The CSV file does have the number of a valid Sales Order in the "Order" field. (In this case: 9)

I also tried to export a CSV and then re-import it, and that failed with the same error.  This is before getting to any validation errors.

As a side question, I'd like to verify all items needed to import full historical sales orders from previous inventory system, and whether there is a certain order that must be followed.  I believe we need, in the following order:
1. Sales Orders
2. Sales Order Line Items
3. Sales Order Allocations
4. Sales Order Shipments

Thanks for any help!

### Steps to Reproduce

1.  Create CSV of Sales Order Line Items, based on an exported file.
2. Attempt to import the new file.
3. Receive error.

### Expected behaviour

I would expect the file to be "accepted" and to either show the data to be imported, or show any validation errors.

### Deployment Method

- [ ] Docker
- [X] Bare metal

### Version Information

# Version Information:
InvenTree-Version: 0.12.8
Django Version: 3.2.19
Commit Hash: a7487ff8
Commit Date: None
Commit Branch: stable
Database: mysql
Debug-Mode: False
Deployed using Docker: False
Platform: Linux-5.15.0-84-generic-x86_64-with-glibc2.35
Installer: PKG
Target: ubuntu:20.04
Active plugins: [{'name': 'InvenTreeBarcode', 'slug': 'inventreebarcode', 'version': '2.0.0'}, {'name': 'InvenTreeCoreNotificationsPlugin', 'slug': 'inventreecorenotificationsplugin', 'version': '1.0.0'}]

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

### Relevant log output

```shell
Line number: 1 - SalesOrderLineItem has no order.
100' Extension Cord, , Orange - Inside/Outside, 1, , 1, SO-000002, , , 2009-6-19, 9, 1, USD, 12.87, 1

Traceback (most recent call last):
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/resources.py", line 644, in import_row
diff = self.get_diff_class()(self, original, new)
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/resources.py", line 204, in __init__
self.left = self._export_resource_fields(resource, instance)
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/resources.py", line 225, in _export_resource_fields
return [resource.export_field(f, instance) if instance else "" for f in resource.get_user_visible_fields()]
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/resources.py", line 225, in <listcomp>
return [resource.export_field(f, instance) if instance else "" for f in resource.get_user_visible_fields()]
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/resources.py", line 845, in export_field
return field.export(obj)
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/fields.py", line 122, in export
value = self.get_value(obj)
File "/opt/inventree/env/lib/python3.10/site-packages/import_export/fields.py", line 98, in get_value
value = value()
File "/opt/inventree/InvenTree/order/models.py", line 1354, in fulfilled_quantity
query = self.order.stock_items.filter(part=self.part).aggregate(fulfilled=Coalesce(Sum('quantity'), Decimal(0)))
File "/opt/inventree/env/lib/python3.10/site-packages/django/db/models/fields/related_descriptors.py", line 197, in __get__
raise self.RelatedObjectDoesNotExist(
order.models.SalesOrderLineItem.order.RelatedObjectDoesNotExist: SalesOrderLineItem has no order.
```

**Comment by SchrodingersGat** [comment]
Yes the `SalesOrder` items must be imported first - and then the line items. Is this the process you have followed?

**Comment by bhotrock** [comment]
Yes, I brought in the Sales Orders first.  And I've confirmed that the Sales Order that is present in the Line Item import does exist.

**Comment by SchrodingersGat** [comment]
And have you guaranteed that the sales order with ID `<9>` exists? Check URL `/order/sales-order/9/`

**Comment by bhotrock** [comment]
@SchrodingersGat  Thanks for your work on this!  
I attempted to put your changed code into my installation, but originally couldn't get it to work.  It was either an error on my part, OR I think my 12.8 install may have been too far behind your development environment.  (I would just end up with 502 Bad Gateway.)

I was finally able to update my bare metal (Installer based) system to version .13.  I have no idea what the actual correct steps would be to do that, but I'm now on .13.  Yay!  After that update, I applied your patches and the import of the SalesOrderLineItems worked correctly.

Thanks again for your great work!
